x-pack/plugins/observability_ai_assistant/server/service/client/index.ts

```diff
@@ -588,20 +595,30 @@ export class ObservabilityAIAssistantClient {
     messages,
     connectorId,
     signal,
+    responseLanguage,
   }: {
     messages: Message[];
     connectorId: string;
     signal: AbortSignal;
+    responseLanguage: string;
   }) => {
     const response$ = await this.chat('generate_title', {
       messages: [
         {
           '@timestamp': new Date().toISOString(),
           message: {
             role: MessageRole.User,
-            content: messages.slice(1).reduce((acc, curr) => {
-              return `${acc} ${curr.message.role}: ${curr.message.content}`;
-            }, 'You are a helpful assistant for Elastic Observability. Assume the following message is the start of a conversation between you and a user; give this conversation a title based on the content below. DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes. This title is shown in a list of conversations to the user, so title it for the user, not for you. Here is the content:'),
+            content: messages.slice(1).reduce(
+              (acc, curr) => {
+                return `${acc} ${curr.message.role}: ${curr.message.content}`;
+              },
+              dedent(`You are a helpful assistant for Elastic Observability.
+            Assume the following message is the start of a conversation between you and a user;
+            give this conversation a title based on the content below.
+            DO NOT UNDER ANY CIRCUMSTANCES wrap this title in single or double quotes.
+            This title is shown in a list of conversations to the user, so title it for the user, not for you.
+            Please create the title in ${responseLanguage}. Here is the content: `)
+            ),
```

Comment:
I think the prompt should be "the following messages" because we actually include the whole conversation, we only replace the system message (`slice(1)`), so I believe the `reduce` makes sense.

I can add the pluralization though.